system/sensord/sensors/lsm6ds3_accel.py

```diff
@@ -38,35 +55,66 @@ def init(self):
     else:
       self.source = log.SensorEventData.SensorSource.lsm6ds3
 
+    # reset chip before init
+    self.write(self.LSM6DS3_ACCEL_I2C_REG_CTRL3_C, 0b1)
+    time.sleep(0.1)
```

Comment:
reminder to remove this since it will also reset the gyro settings

handled it properly here: https://github.com/commaai/openpilot/pull/35872